src/main/java/edu/cornell/kfs/fp/batch/service/impl/AccountingXmlDocumentDownloadAttachmentServiceImpl.java

```diff
@@ -76,8 +81,8 @@ public Attachment createAttachmentFromBackupLink(Document document,
     
     protected boolean isLinkUrlValidForGroupCode(AccountingXmlDocumentBackupLink accountingXmlDocumentBackupLink, Collection<WebServiceCredential> credentials) {
         if (CollectionUtils.isEmpty(credentials)) {
-            LOG.debug("isLinkUrlValidForGroupCode, no credential values for group code, so is valid");
-            return true;
+            LOG.debug("isLinkUrlValidForGroupCode, no credential values for group code, so is invalid");
```

Comment:
should this be a LOG.error?

Reply:
Good catch, I agree that it makes sense to change the log level of this statement. I will make that change.